x-pack/test/functional/services/machine_learning/job_wizard_common.ts

```diff
@@ -211,24 +321,12 @@ export function MachineLearningJobWizardCommonProvider({ getService }: FtrProvid
       });
     },
 
-    async activateDedicatedIndexSwitch() {
-      if ((await this.getDedicatedIndexSwitchCheckedState()) === false) {
-        await testSubjects.clickWhenNotDisabled('mlJobWizardSwitchUseDedicatedIndex');
-      }
-    },
-
-    async setModelMemoryLimit(modelMemoryLimit: string) {
-      await testSubjects.setValue('mlJobWizardInputModelMemoryLimit', modelMemoryLimit, {
-        withKeyboard: true,
-      });
-    },
-
     async createJobAndWaitForCompletion() {
       await testSubjects.clickWhenNotDisabled('mlJobWizardButtonCreateJob');
       await retry.waitForWithTimeout(
         'job processing to finish',
         5 * 60 * 1000,
-        async () => (await testSubjects.exists('mlJobWizardButtonCreateJob')) === false
+        async () => await testSubjects.exists('mlJobWizardButtonRunInRealtime')
```

Comment:
As above, better to use `mlJobWizardButtonRunInRealTime` with uppercase `T` for consistency.

Reply:
Changed in c3ef36f